features/support/PaymentGateway.js

```diff
@@ -0,0 +1,20 @@
+const Stripe = require('stripe')
+const http = require('http')
+
+module.exports = class PaymentGateway {
+  constructor({ type, secretKey }) {
+    this.type = type;
+    if(this.type === 'Stripe') {
+      this.stripe = Stripe(secretKey);
+    }
+  }
+
+  products() {
+    return new Promise((resolve, reject) => {
+      this.stripe.products.list({limit: 100}, (err, products) => {
```

Comment:
Is this `limit: 100` from Stripe?

Reply:
Yea, I figured 100 was a better default.